lib/utilshare/accounts/dwolla.ex

```diff
@@ -0,0 +1,83 @@
+defmodule Utilshare.Accounts.Dwolla do
+
+  def token do
+    GenServer.call(Utilshare.TokenManager, {:get, :token})
+  end
+
+  def authenticate_server() do
+    body = {:form, 
+    [
+      client_id: Application.get_env(:utilshare, :key),
+      client_secret: Application.get_env(:utilshare, :secret),
+      grant_type: "client_credentials",
+    ]}
+    case HTTPoison.post("https://sandbox.dwolla.com/oauth/v2/token", body, %{"Content-type" => "application/x-www-form-urlencoded"}) do
+      {:ok, %HTTPoison.Response{status_code: 200, body: body}} ->
```

Comment:
Is there a way to avoid checking the 200 status code? So that this matches any 2xx